tools/pyclient/dev/dev.py

```diff
@@ -0,0 +1,85 @@
+#///////////////////////////////////////////////////////////////////////////////
+# FILE: dev.py
+# AUTHOR: David Ruvolo
+# CREATED: 2023-05-22
+# MODIFIED: 2023-07-21
+# PURPOSE: development script for initial testing of the py-client
+# STATUS: ongoing
+# PACKAGES: pandas
+# COMMENTS: Designed to interact with the schema "pet store".
+#///////////////////////////////////////////////////////////////////////////////
+
+import pandas as pd
+
+from tools.pyclient.src.molgenis_emx2_pyclient import Client
+
+# connect and sign in
+db = Client('https://emx2.dev.molgenis.org/')
+db.sign_in('', '')
+
+# check status
+db.status()
+
+# get data
+data = db.get(schema='', table='') # run without specifying target
+data = db.get(schema = 'pet store', table='') # run without specifying table
+data = db.get(schema = 'pet store', table='Pet') # get Pets
+data
+
+data = db.get(schema = 'pet store', table='Pet', as_df=True) # get Pets

```

Comment:
would it not make sense to have a db.getDf vs db.getXyz (I don't know what alternative is yet because didn't review that part yet). Based on next lines I suppose JSON

Reply:
Since pandas is used to parse the response from the `<schema>/api/csv/<table>`, we wanted to give the user the option (in the first instance) to return the data as a DataFrame object. But this is something we haven't discussed yet.  I'll add it to the list.